### PR TITLE
Skip post_migrate when running tests

### PR DESCRIPTION
**Pull request by netpok**
## Summary
This PR skips post migrate hooks when application is running unit tests. This should not cause any problems but it's unnecessary to generate the helpers hundred times. Also test should not have a sideeffect.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist
- [x] Add a CHANGELOG.md entry
- [x] Code style has been fixed via `composer fix-style`

### CODE REVIEW

File: src/IdeHelperServiceProvider.php [hunk 1]
```diff
@@ -35,7 +34,7 @@ class IdeHelperServiceProvider extends ServiceProvider implements DeferrableProv
      */
     public function boot()
     {
-        if ($this->app['config']->get('ide-helper.post_migrate', [])) {
+        if (!$this->app->runningUnitTests() && $this->app['config']->get('ide-helper.post_migrate', [])) {
```
Comment (mfn) [inline src/IdeHelperServiceProvider.php:37]: Will this work with Lumen? 😬

Comment (netpok) [reply src/IdeHelperServiceProvider.php:37]: It should: https://github.com/laravel/lumen-framework/blob/8.x/src/Application.php#L896

Comment (mfn) [reply src/IdeHelperServiceProvider.php:37]: Thank you, LGTM!

Could you please resolve the merge conflicts? Don't see any blockerS!

**Review by mfn — APPROVED** [review]
Thank you!

**Pull request merged**